tests/dummy/app/styles/app.css

```diff
@@ -170,3 +191,20 @@ tr.dragSortItem.-placeholderAfter td {
 table .the-item {
   margin: 0;
 }
+
+/* Occlusion Rendering */
+.dragSortList.-isLazyRenderActive {
+  border: 3px solid #2f7ada;
+}
```

Comment:
same here, we can use variable for color

Reply:
This is a plain css file and not a preprocessor like sass/less. Of course, we do have css variables, but its hard to maintain and concerned about the backward-compatibility. Moreover this is under `test/` folder which is to test the addon during development phase and will not have any impact on the consuming applications.